token/program/src/instruction.rs

```diff
@@ -258,48 +258,119 @@ impl TokenInstruction {
         let mut output = vec![0u8; size_of::<TokenInstruction>()];
         match self {
             Self::InitializeMint { amount, decimals } => {
-                output[0] = 0;
+                let mut output_len = 0;
```

Comment:
Can we move these outside of the match rather than duplicated in each?

`let mut output_len = 0;`
`output.truncate(output_len);`

Reply:
This change has been made in the most recent push